team_delta.js

```diff
@@ -1 +1,26 @@
 // Fetch all routes for the airline and insert into #delta
+const getFlight = async (from, to, resourse) => {
+  const response = await fetch(resourse);
+  const dataJson = await response.json();
+  const searchEl = document.createElement("div");
+  const logoEl = document.createElement("img");
+  const deltaEl = document.querySelector("#delta");
+  const routeEl = document.createElement("p");
+  const priceEl = document.createElement("p");
+  logoEl.setAttribute("src", "https://logo.clearbit.com/delta.com");
+  routeEl.innerText = `Flight from ${from} to ${to}`;
```

Comment:
Make sure you check that you get the parameters correctly (e.g. they are not empty).